### DISCUSSION TITLE
Show on poster if Movie is available in UHD only

### DISCUSSION THREAD

**Discussion by lerdelyi** [sct/overseerr#3884] [Feature Requests]
Currently, from what I can see, if a Movie is available in any format, other than 4K (UHD) it will have a check-mark on the poster, but if only the 4K version is available no such check mark exists. It would be useful to have either the same checkmark or a different mark on the poster (maybe like 4K) or something to indicate that the current content is available but only in 4K.

Current version: 1.33.2 being used.

**Comment by stevendarby** [comment]
@lerdelyi this issue was raised and closed a while back: https://github.com/sct/overseerr/issues/3010

Perhaps a few extra voices and suggestions could get it re-opened and implemented?
